requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Used to install pinned dependencies
 # Useful for dev/test jobs caches
 # Must be kept in sync with setup.py
-torch == 2.2.0 # This is what is installed in CI today
+torch ~= 2.2 # This is what is installed in CI today
```

Comment:
I think we also have an `install_requires=["torch >= 2.1"],` in setup.py that we should probably also change

Reply:
I am not seeing that here:

```
install_requires=get_requirements()
```

Reply:
ah I see, I was looking in the wrong repo. We may want to update the others to be consistent after this:

https://github.com/foundation-model-stack/fms-fsdp/blob/main/setup.py#L12

and

https://github.com/foundation-model-stack/foundation-model-stack/blob/main/setup.py#L12

Reply:
Will open PR there as well shortly